payroll-processor-functions/src/payroll-processor-functions/local.settings.json.sample

```diff
@@ -0,0 +1,11 @@
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureTableStorage": "UseDevelopmentStorage=true"
```

Comment:
This is a convention used for all the injected `[Table]`/`[Queue]` attributes in the functions.

They automatically source their connection from this settings property.

Reply:
So I had to use `"AzureWebJobsStorage": "UseDevelopmentStorage=true"` as an additional key in this configuration.

Reply:
You know what, that's actually what I have in my local settings - I need to sync that back to the sample!

Reply:
I think this still needs sync'd

Reply:
This was updated but GH isn't updating this comment thread with the latest code... see below.